index.yaml

```diff
@@ -132,8 +132,19 @@ sources:
     homepage: https://raw-data.gitlab.io/post/malsilo_2.1/
     checksum: true
 
+  stamus/lateral:
+    summary: Lateral movement rules
+    description: |
+      Suricata ruleset specifically focused on detecting lateral
+      movement in Microsoft Windows environments by Stamus Networks
+    vendor: Stamus Networks
+    min-version: 6.0.6
+    license: GPLv3
```

Comment:
License confusion.  The `LICENSE` file is GPLv3, however the rules file is MIT license. If sticking with the GPLv3, also note the difference between `GPL-3.0-only` and `GPL-3.0-or-later` as I'd like to move to SPDX license identifiers in this index.

Reply:
fixed